core/codegen/tests/ui-fail-nightly/from_param.stderr

```diff
@@ -0,0 +1,57 @@
+error: named structs are not supported
+ --> tests/ui-fail-stable/from_param.rs:4:1
+  |
+4 | / struct Foo1 {
+5 | |     a: String
+6 | | }
+  | |_^
+
+error: [note] error occurred while deriving `FromParam`
+ --> tests/ui-fail-stable/from_param.rs:3:10
+  |
+3 | #[derive(FromParam)]
+  |          ^^^^^^^^^
+  |
+  = note: this error originates in the derive macro `FromParam` (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error: named structs are not supported
+ --> tests/ui-fail-stable/from_param.rs:9:1
+  |
+9 | struct Foo2 {}
+  | ^^^^^^^^^^^^^^
+
+error: [note] error occurred while deriving `FromParam`
+ --> tests/ui-fail-stable/from_param.rs:8:10
+  |
+8 | #[derive(FromParam)]
+  |          ^^^^^^^^^
+  |
+  = note: this error originates in the derive macro `FromParam` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
It's surprising that this isn't an actual `note`, since they exist on nightly. Is this being tested? Is this the output that's actually being produced?

Reply:
My bad. I supposed that it would produce the same output. Just tested and fixed it.